sample/ClassComponents.fs

```diff
@@ -0,0 +1,74 @@
+﻿module ClassComponents
+
+open Lit
+open Fable.Core
+open Fetch
+
+type MouseController =
+    inherit ReactiveController
+    abstract x: float
+    abstract y: float
+type Post = {| usedId: int; id: int; title: string; body: string |}
+
+[<Import("MouseController", from="./controllers.js"); Emit("new $0($1)")>]
+let private createMouseController (host: LitElement): MouseController = jsNative
```

Comment:
If you need the constructor, instead of using an interface and a separate function, it's probably easier to declare a class:

```fsharp
[<ImportMember("./controllers.js")>]
type MouseController(host: ReactiveControllerHost) =
    inherit ReactiveController(host)
    member _.x: float = jsNative
    member _.y: float = jsNative
```

Reply:
AH I just copied this from the controller sample we had already on the branch I will remove it before I finish this